newIDE/app/src/MainFrame/Preferences/PreferencesDialog.js

```diff
@@ -225,6 +226,14 @@ const PreferencesDialog = ({ onClose }: Props) => {
               label={<Trans>Auto-save project on Preview</Trans>}
             />
           </Line>
+          <Line>
+            <Toggle
+              onToggle={(e, check) => setAutoOpenMostRecent(check)}
+              toggled={values.autoOpenMostRecent}
+              labelPosition="right"
+              label={<Trans>Auto-open the most recent project.</Trans>}
```

Comment:
nit: `Automatically re-open the project edited during last session` (do period at the end for consistency with other settings).